k8s/edi/uwsgi.ini

```diff
@@ -3,5 +3,5 @@ module = legion.edi.wsgi
 http = 0.0.0.0:5000
 uid = www-data
 gid = www-data
-cheaper = 2
 processes = 16
+workers = 2
```

Comment:
Why did you add the `workers` option?
If I understand correctly, `workers` is synonim of the `processes`. `edi` server will have only 2 workers after your changes.

Reply:
Agree